lms/resources/_js_config/__init__.py

```diff
@@ -302,6 +302,9 @@ def enable_toolbar_grading(self, students, score_maximum=None):
         toolbar_config = self._get_toolbar_config()
 
         toolbar_config["gradingEnabled"] = True
+        toolbar_config[
```

Comment:
I reckon this makes sense here, while configuring the grading bar in the first place.